.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,20 @@
+# publish npm package when a change to version in package.json is pushed to master branch
+# more info here: https://github.com/marketplace/actions/npm-publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
```

Comment:
I think this looks fine but should we have a step for running the tests?

Reply:
We'll tackle this in the future, just having the automatic publish step done is a nice improvement.